flang/lib/Lower/Bridge.cpp

```diff
@@ -2029,9 +2031,30 @@ class FirConverter : public Fortran::lower::AbstractConverter {
   void handleLocalitySpecs(const IncrementLoopInfo &info) {
     Fortran::semantics::SemanticsContext &semanticsContext =
         bridge.getSemanticsContext();
-    for (const Fortran::semantics::Symbol *sym : info.localSymList)
+    Fortran::lower::omp::DataSharingProcessor dsp(
+        *this, semanticsContext, getEval(),
+        /*useDelayedPrivatization=*/true, localSymbols);
+    fir::LocalitySpecifierOperands privateClauseOps;
+    auto doConcurrentLoopOp =
+        mlir::dyn_cast_if_present<fir::DoConcurrentLoopOp>(info.loopOp);
+    bool useDelayedPriv =
```

Comment:
If I am readying this correctly, a TODO message is needed for using locality specifiers without delayed privatization staging.

Reply:
Similar to delayed privatization in OpenMP, I am planning to promote this to the non-staging flag (which is enable by default) once we have the stack merged. Just to make it less costly to hide it back behind the staging flag if we discover problems in the wild. Added a TODO.

Reply:
Thanks

Reply:
Ahh sorry I wasn't clear. I meant the TODO macro e.g. `TODO(loc, "do-concurrent locality specifiers")`

Reply:
Just to make sure I understand. Under what condition would the todo trigger? My plan to use the non-staging flag very soon (probably after merging this PR stack).

Reply:
Ahh that's okay then :)